src/constraint_system/logic.rs

```diff
@@ -63,6 +63,7 @@ impl TurboComposer {
         let a_bits: Vec<_> = a_bit_iter.skip(256 - num_bits).collect();
         let b_bit_iter = BitIterator8::new(self.witnesses[&b].to_bytes());
         let b_bits: Vec<_> = b_bit_iter.skip(256 - num_bits).collect();
+        // panic!("a bits: {:?}\nb bits: {:?}", a_bits, b_bits);
```

Comment:
nit?

Reply:
Good catch! Needed this for debugging zelbet, will remove it